content/en/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,19 @@
+---
+title: affinity
+id: affinity
+date: 2019-01-11
+full_link: /docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/
+short_description: >
+  Node affinity is a set of rules used by the scheduler to determine where a pod can be placed.
+aka:
+tags:
+- core object
+- fundamental
+---
+Node affinity is a set of rules used by the scheduler to determine where a pod can be placed.
+
+<!--more-->
+
+The rules are defined using the familiar concepts of custom labels on {{< glossary_tooltip term_id="node" text="nodes">}} 
```

Comment:
Because there are two kinds of affinity, I'd write:
```suggestion
The rules are defined using the Kubernetes {{< glossary_tooltip term_id="label" text="labels">}},
```